src/Umbraco.Web.UI.Client/src/views/webhooks/logs.controller.js

```diff
@@ -22,9 +26,14 @@
     }
 
     function loadLogs() {
-      return webhooksResource.getLogs()
+      const take = vm.pagination.pageSize;
+      const skip = (vm.pagination.pageNumber - 1) * take;
+
+      return webhooksResource.getLogs(skip, take)
         .then(data => {
           vm.logs = data.items;
+          vm.pagination.totalPages = Math.ceil(data.totalItems/vm.pagination.pageSize);
```

Comment:
Doesn't resource return a `totalPages` property simillar to users for example?
https://github.com/umbraco/Umbraco-CMS/blob/v13/dev/src/Umbraco.Web.UI.Client/src/views/users/views/users/users.controller.js#L730

Reply:
Not for this endpoint, it uses skip/take for pagination so `totalPages` always returns `0`

Reply:
Okay, perhaps it should have a `getPagedLogs()`
https://github.com/umbraco/Umbraco-CMS/blob/v13/dev/src/Umbraco.Web.UI.Client/src/common/resources/webhooks.resource.js#L39

`GetLogs()` returns a `PagedResult<T>`:
https://github.com/umbraco/Umbraco-CMS/blob/v13/dev/src/Umbraco.Web.BackOffice/Controllers/LogViewerController.cs#L94

which has `totalPages` property:
https://github.com/umbraco/Umbraco-CMS/blob/contrib/src/Umbraco.Core/Models/PagedResultOfT.cs#L11
https://github.com/umbraco/Umbraco-CMS/blob/contrib/src/Umbraco.Core/Models/PagedResult.cs#L34-L35